datasets/cil-gdpcir/dataset.yaml

```diff
@@ -0,0 +1,55 @@
+name: cil_gdpcir
+image: ${{ args.registry }}/pctasks-task-base:latest
+
+args:
+- registry
+
+code:
+  src: ${{ local.path(./cil_gdpcir.py) }}
```

Comment:
This isn't used anywhere since it's all premade stac items...you can remove this block and the code file